chromadb/test/test_api.py

```diff
@@ -1207,6 +1207,11 @@ def test_invalid_id(client):
         collection.add(embeddings=[0, 0, 0], ids=[1], metadatas=[{}])
     assert "ID" in str(e.value)
 
+    # Upsert with an empty id
```

Comment:
test api is a kitchen sink we should leave untouched. Please move the test out of this file.

I like the basic unit tests but would prefer to see basic tests that cover cases like

- No ids supplied - and the generated ids when retrieved with get() return what you expect
- Partial ids supplied fails
- If you supply ids, the expected result is returned

Reply:
added